src/server/lang.js

```diff
@@ -0,0 +1,43 @@
+import path from "path"
+import {readdir} from "fs"
+
+import {addLocaleData} from "react-intl"
+import fr from "react-intl/locale-data/fr"
+import en from "react-intl/locale-data/en"
```

Comment:
Shouldn't these be dynamically required from `loadTranslations`? Otherwise, I don't see how you can load a full directory by iterating over whatever's defined in it.